xmppserver/src/main/webapp/muc-room-clear-chat.jsp

```diff
@@ -0,0 +1,96 @@
+<%@ page contentType="text/html; charset=UTF-8" %>
+<%--
+  -
+  - Copyright (C) 2004-2008 Jive Software, 2017-2024 Ignite Realtime Foundation. All rights reserved.
```

Comment:
The copyright needs not include Jive Software. Please replace it with:
> Copyright (C) 2024 Ignite Realtime Foundation.

If any of the files that you edit have a copyright notice that doesn't include the current year for the Ignite Realtime Foundation, please update that, too.

Reply:
Done.